distributed/protocol/tests/test_protocol.py

```diff
@@ -313,10 +335,9 @@ def test_dumps_loads_Serialized():
 
 def test_maybe_compress_memoryviews():
     np = pytest.importorskip("numpy")
-    pytest.importorskip("lz4")
     x = np.arange(1000000, dtype="int64")
     compression, payload = maybe_compress(x.data)
-    assert compression == "lz4"
+    assert compression in {"lz4", "snappy", "zstd", "zlib"}
```

Comment:
I think I would be sad if we used `zlib` by default in any configuration.  I'll bet that it's faster to just send data uncompressed over the network.

Reply:
Ah, you're right. I misread the code; default compression is lz4 -> snappy -> None.
I've amended the tests and added a specific test for the priority order.